scripts/sign.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+PROGRAM_NAME="${0##*/}"
+
+# Read input args
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -h|--help) HELP=true ;;
+        -d|--directory) content_dir="$2"; shift ;;
+        -s|--secret-key) secret_key_file=$2; shift ;;
+        -p|--public-key) public_key_file=$2; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+if [ "$HELP" = true ] ; then
+    echo "Usage: $PROGRAM_NAME [options]"
+    echo ""
+    echo "Signs all .json files within input directory"
+    echo ""
+    echo "Options:"
+    echo "-h, --help                Show this help message and exit."
+    echo "-d, --directory <path>    Input folder containing .json files to sign."
+    echo "-s, --secret-key <path>   Path to secret key used for signing."
+    echo "-s, --public-key <path>   Path to public key used for signing."
```

Comment:
Might be useful to add meaningful defaults and communicate them here, or add a bit of text denoting if a param is required or not. (This is further to my other comment).